lib/endpoints/class-wp-rest-comments-controller.php

```diff
@@ -640,6 +640,18 @@ protected function prepare_links( $comment ) {
 			);
 		}
 
+		// Only grab one comment to verify the comment has children.
+		$comment_children = $comment->get_children( array( 'number' => 1 ) );
```

Comment:
That looks like it will save even more execution time and the cache will be a lot smaller 👍! Thank you for the tip.